packages/flutter_tools/gradle/plugins_init.gradle

```diff
@@ -0,0 +1,87 @@
+import java.nio.file.Path
+import java.nio.file.Paths
+
+allprojects {
+    apply plugin: FlutterAarPlugin
+}
+
+/** Adds in a compileOnly dependency on flutterJar. */
```

Comment:
I'm confused by the context of this comment.  Should this comment go on top of, or within a procedure somewhere?